packages/pigeon/platform_tests/shared_test_plugin_code/lib/integration_tests.dart

```diff
@@ -549,7 +549,7 @@ void runPigeonIntegrationTests(TargetGenerator targetGenerator) {
         (WidgetTester _) async {
       final HostIntegrationCoreApi api = HostIntegrationCoreApi();
 
-      const List<Object?> sentObject = <Object>[7, 'Hello Dart!'];
+      const List<Object?> sentObject = <Object?>[7, 'Hello Dart!', null];
```

Comment:
These two changes are both good things to add, but isn't the code being changed specific to deserialization of nullable fields whose type is classes (which happen to be lists), not actual collection types?

Reply:
Misunderstood. I'll add that.